src/dependencies.jl

```diff
@@ -76,7 +76,7 @@ function clone_all_registries(f::Function, registry_list::Vector{Pkg.RegistrySpe
     end
 end
 
-function get_latest_version!(deps, registries)
+function get_latest_version!(deps, registries; exclude_yanked::Bool = true)
```

Comment:
Think we need to populate this all the way down from `main()` if we want to use it.

Reply:
Let's do #419 first, which should make it a little easier to pass options around.

Reply:
Makes sense to me! I'll review it shortly.